report/2.Analysis/EducationalMiniComputers.tex

```diff
@@ -0,0 +1,30 @@
+\section{Educational mini computers}
+
+Apart from the Arduino board, a lot of other mini computers have been created with educational purposes in mind.
+To figure out why these "mini computers" are so popular for technical learning, some of the most popular ones were analysed.
+This should help to figure out which functionality the HCL programming language should include in order to comply with the intend of the mini computers.
+
+\subsection{Raspberry Pi}
+The Raspberry Pi is one of the most widely known single board computers. 
+It is a complete arm-based computer, and may run anything arm based on it, including both linux and windows based arm operating systems. 
+This first of all makes it a common entry point into the GNU/Linux operating systsem stack, which is generally considered to be very educational in regards to how computers work.
+The default Raspberry Pi distribution ships with both scratch and python, allowing the user to quickly get started with some programming.\cite{RaspberryPi}
+
+On top of this, one of the major selling points to the Raspberry Pi is the GPIO\footnote{general purpose input output} pins. 
+These allow the user to attach all kind of devices to the board, this includes both sensors, lights, cameras and breadboards. 
+The Raspberry Pi allows the user to easily interact with the GPIO pins, creating a great device for both IoT, home automation and robotics.\cite{RaspberryPi}
```

Comment:
* "great" have the same problem as "very".

Reply:
Having a little hard time rephrasing the great and very sentence, do you have any suggestions?

Reply:
* "very" can easily be replaced with "highly". The reason being that "highly" narrows the measurment down to upper and 
  lower which is mondain in this context and completely acceptable.
* "great" just needs to be translated into the actual meaning. Like do we mean "the subject in the context is best suitable for" then suitable could be written instead

Reply:
Check it now. Hope this works 😄